packages/migrate/src/commands/DbPull.ts

```diff
@@ -237,14 +237,16 @@ Set composite types introspection depth to 2 levels
 
             const result = { firstDatasource: config.datasources[0], schema, validationWarning: undefined }
 
-            const hasDriverAdaptersPreviewFeature = (previewFeatures || []).includes('driverAdapters')
+            const hasSchemaEngineeDriverAdaptersPreviewFeature = (previewFeatures || []).includes(
+              'schemaEngineDriverAdapters',
+            )
             const validationWarning = `Without the ${bold(
-              'driverAdapters',
+              'schemaEngineDriverAdapters',
             )} preview feature, the schema introspected via the ${bold('--local-d1')} flag will not work with ${bold(
               '@prisma/client',
             )}.`
 
-            if (hasDriverAdaptersPreviewFeature) {
+            if (hasSchemaEngineeDriverAdaptersPreviewFeature) {
```

Comment:
I know it not specific to this PR. Does anyone know why we went for a `if.. else` instead of a one liner?
like:

```TS
return hasSchemaEngineeDriverAdaptersPreviewFeature ?   result : { ...result, validationWarning };
```

I personally think it's shorter and not necessarily harder to read

Reply:
Thanks for the response. 
Again no biggies, I don't think it's wrong or anything, I just try to understand why we did it in the first place. 🙂

> I don't think it matters much one way or the other.
Fair, the reason I think it matters, is because, as it stands the code is taking a few unnecessary lines and is just about as wide. (counting the if statement indent)

> It's certainly harder to read in the GitHub PR review UI:
I don't think anybody is using the GitHub PR review comment UI to read the entire code. This matters when we open the code in a code editor, IDE or even when opening the entire file from GitHub.

To be honest, if I were to write the existing code in the comment, it would be just as hard to read.